includes/Modules/Subscribe_With_Google/ManagePosts.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Subscribe_With_Google\ManagePosts
+ *
+ * @package   Google\Site_Kit\Modules\Subscribe_With_Google
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Subscribe_With_Google;
+
+/**
+ * Adds SwG column to Manage Posts page.
+ */
+final class ManagePosts {
+
+	/** Adds WordPress filters. */
+	public function __construct() {
+		add_filter( 'manage_posts_columns', array( __CLASS__, 'manage_posts_columns' ) );
+		add_action( 'manage_posts_custom_column', array( __CLASS__, 'manage_posts_custom_column' ), 1, 2 );
+	}
+
+	/**
+	 * Filters manage posts columns.
+	 *
+	 * @param string[] $columns Columns to filter.
+	 * @return string[]
+	 */
+	public static function manage_posts_columns( $columns ) {
```

Comment:
Another general observation, in Site Kit we don't use static methods unless absolutely necessary. Classes should typically be instantiated and have their own state. While that may technically not matter much in production code, it facilitates separation of concerns and easier testing.